dataland-frontend/src/components/forms/CreateEuTaxonomyForNonFinancials.vue

```diff
@@ -0,0 +1,343 @@
+<template>
+  <Card class="col-12 page-wrapper-card p-3">
+    <template #title
+      ><span data-test="pageWrapperTitle">
+        {{ editMode ? "Edit" : "Create" }} EU Taxonomy Dataset for a Non-Financial Company/Service</span
+      ></template
+    >
+    <template #content>
+      <div v-if="waitingForData" class="d-center-div text-center px-7 py-4">
+        <p class="font-medium text-xl">Loading Eu Taxonomy For Non Financials data...</p>
+        <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
+      </div>
+      <div v-else class="grid uploadFormWrapper">
+        <div id="uploadForm" class="text-left uploadForm col-9">
+          <FormKit
+            v-model="companyAssociatedEuTaxonomyDataForNonFinancials"
+            :actions="false"
+            type="form"
+            :id="formId"
+            :name="formId"
+            @submit="postEuTaxonomyForNonFinancialsData"
+            @submit-invalid="checkCustomInputs"
+          >
+            <FormKit type="hidden" name="companyId" :model-value="companyID" disabled="true" />
```

Comment:
Why `disabled`?

Reply:
No idea. I tested with and without and has no effect on the Reporting period date field.
Removed. Done.

Reply:
Reverted. I did a bit of digging into the matter. This is a hidden input and by disabling it we prevent it from being sent with the form. It's value `companyID` is used in other parts of the code.